src/main.ts

```diff
@@ -56,10 +57,10 @@ export async function runSorald(
     core.info('Attempting repairs');
     for (const [ruleKey, violationSpecs] of keyToSpecs.entries()) {
       core.info(`Repairing violations of rule ${ruleKey}: ${violationSpecs}`);
-      const statsFile = path.join(source.toString(), `${ruleKey}.json`);
+      const statsFile = path.join(sourceAbsPath.toString(), `${ruleKey}.json`);
```

Comment:
`sourceAbsPath` will always be a string if [`path.resolve`](https://nodejs.org/api/path.html#path_path_resolve_paths) doesn't throw an exception so you need not call `toString`.

Reply:
Ah, yes, that is true. I just did a variable replace here, replacing all uses of `source` with `sourceAbsPath`.